<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>제품ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>제품이름:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>제품가격:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prodID">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="prodName">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>190</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="prodPrice">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>190</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>입력</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>수정</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>삭제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>160</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>721</width>
      <height>301</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>20</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>43</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addProduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateProduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>528</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeProduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>getProduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>516</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addProduct()</slot>
  <slot>updateProduct()</slot>
  <slot>removeProduct()</slot>
  <slot>getProduct()</slot>
 </slots>
</ui>
